packages/manager/modules/pci/src/projects/project/project.controller.js

```diff
@@ -45,9 +46,31 @@ export default class ProjectController {
     this.projectQuotaAboveThreshold = this.quotas.find(
       (quota) => quota.quotaAboveThreshold,
     );
+
+    const featuresName = ProjectController.findFeatureToCheck(ACTIONS);
+    this.ovhFeatureFlipping
```

Comment:
**nitpick:** Maybe we can add a `return` statement here.

Reply:
return statement in $onInit method? why and what are the advantages?

Reply:
Not mandatory in this context here (`$onInit`), but usually when it comes to deal with Promises, it still a good habit to use a `return` statement. That's why here I'm adding this comment as a `nitpick`.